packages/ui/src/components/Form/dataFormat/DataFormatEditor.tsx

```diff
@@ -58,235 +32,69 @@ export const DataFormatEditor: FunctionComponent<DataFormatEditorProps> = (props
-          {...(activeItem && { 'aria-activedescendant': activeItem })}
-          role="combobox"
-          isExpanded={isOpen}
-          aria-controls="select-typeahead-listbox"
-        />
-
-        <TextInputGroupUtilities>
-          {!!inputValue && (
-            <Button
-              data-testid="clear-input-value"
-              variant="plain"
-              onClick={() => {
-                setSelected('');
-                setInputValue('');
-                setFilterValue('');
-                handleOnChange('', {});
-                textInputRef?.current?.focus();
-              }}
-              aria-label="Clear input value"
-            >
-              <TimesIcon aria-hidden />
-            </Button>
-          )}
-        </TextInputGroupUtilities>
-      </TextInputGroup>
-    </MenuToggle>
-  );
-
   return (
-    dataFormatCatalogMap && (
-      <div className="dataformat-metadata-editor">
-        <Card isCompact={true} isExpanded={isExpanded}>
-          <CardHeader onExpand={() => setIsExpanded(!isExpanded)}>
-            <CardTitle>Data Format</CardTitle>
-          </CardHeader>
-          <CardExpandableContent>
-            <CardBody data-testid={'dataformat-config-card'}>
-              <Select
-                id="typeahead-select"
-                isOpen={isOpen}
-                selected={selected}
-                onSelect={onSelect}
-                onOpenChange={() => {
-                  setIsOpen(false);
-                }}
-                toggle={toggle}
-              >
-                <SelectList id="select-typeahead-listbox">
-                  {selectOptions.map((option, index) => (
-                    <SelectOption
-                      key={option.value as string}
-                      description={option.description}
-                      isFocused={focusedItemIndex === index}
-                      className={option.className}
-                      data-testid={`dataformat-dropdownitem-${option.value}`}
-                      onClick={() => setSelected(option.children as string)}
-                      id={`select-typeahead-${option.value.replace(' ', '-')}`}
-                      {...option}
-                      value={option.children}
-                    />
-                  ))}
-                </SelectList>
-              </Select>
-              {dataFormat && (
-                <MetadataEditor
-                  key={dataFormat.model.name}
-                  data-testid="dataformat-editor"
-                  name="dataformat"
-                  schema={dataFormatSchema}
-                  metadata={dataFormatModel}
-                  onChangeModel={(model) => handleOnChange(dataFormat.model.name, model)}
-                />
-              )}
-            </CardBody>
-          </CardExpandableContent>
-        </Card>
-      </div>
-    )
+    <div className="dataformat-metadata-editor">
+      <Card isCompact={true} isExpanded={true}>
```

Comment:
We are missing the expandable functionality :eyes: 

```suggestion
      <Card isCompact isExpanded={isExpanded}>
```